src/errors/HttpRequestError.ts

```diff
@@ -0,0 +1,27 @@
+type ObjectLike = Record<string, any>;
```

Comment:
Please prefer using more strictier `unknown` type when actual one couldn't be determined instead of `any`.